src/modules/secrets/secrets.controller.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  ParseIntPipe,
+  Patch,
+  Post,
+  Put,
+} from '@nestjs/common';
+import { SecretsService } from './secrets.service';
+import { CreateSecretsDto } from './dtos/create-secrets.dto';
+import { ApiTags } from '@nestjs/swagger';
+import { UpdateSecretsDto } from './dtos/update-Secrets.dto';
+
+@ApiTags('secrets')
+@Controller('secrets')
+export class SecretsController {
+  constructor(private readonly secretsService: SecretsService) {}
+
+  @Post(':accountId/secrets')
+  async createSecret(
+    @Param('accountId', ParseIntPipe) accountId: number,
+    @Body() createSecretDto: CreateSecretsDto,
+  ) {
+    return this.secretsService.createSecret(createSecretDto,accountId);
+  }
+
+  @Get(':accountId/secrets')
```

Comment:
This  GET route here looks like _"/secrets/:accountId/secrets"._ 
You can tell that **secrets** is redundant here.

Please try to follow REST conventions when constructing the routes.

The same analogy can be made for other routes in here, no need to replicate a resource name in a route.